ocs_ci/helpers/managed_services.py

```diff
@@ -219,3 +222,116 @@ def verify_osd_distribution_on_provider():
         assert (
             osd_count == int(size) / 4
         ), f"Zone {zone} does not have {size/4} osd, but {osd_count}"
+
+
+def verify_storageclient(
+    storageclient_name=None, namespace=None, provider_name=None, verify_sc=True
+):
+    """
+    Verify status, values and resources related to a storageclient
+
+    Args:
+        storageclient_name (str): Name of the storageclient to be verified. If the name is not given, it will be
+            assumed that only one storageclient is present in the cluster.
+        namespace (str): Namespace where the storageclient is present.
+            Default value will be taken from ENV_DATA["cluster_namespace"]
+        provider_name (str): Name of the provider cluster to which the storageclient is connected.
+        verify_sc (bool): True to verify the storageclassclaims and storageclasses associated with the storageclient.
+
+    """
+    storageclient_obj = OCP(
+        kind=constants.STORAGECLIENT,
+        namespace=namespace or config.ENV_DATA["cluster_namespace"],
+    )
+    storageclient = (
+        storageclient_obj.get(resource_name=storageclient_name)
+        if storageclient_name
+        else storageclient_obj.get()["items"][0]
+    )
+    storageclient_name = storageclient["metadata"]["name"]
+    provider_name = provider_name or config.ENV_DATA.get("provider_name", "")
+    endpont_actual = get_storage_provider_endpoint(provider_name)
```

Comment:
Just a tiny note - change the name from `endpont_actual` to `endpoint_actual`.

Reply:
Done